zerver/lib/test_classes.py

```diff
@@ -117,6 +118,20 @@ class EmptyResponseError(Exception):
     pass
 
 
+class ZulipClientHeaders(TypedDict, total=False):
+    """
+    This is a subset of headers being used with the test client very specifically in
+    the form of `client_foo(**headers)`, existing only for the purpose of guaranteeing
+    that the keyword arguments have compatible types.
+    """
+
+    content_type: str
+    stream_name: str
+    HTTP_POST: str
+    HTTP_HOST: str
+    HTTP_AUTHORIZATION: str
+
+
```

Comment:
I am using `ZulipClientHeaders` for now. But would it be clearer to name it `TestClientHeaders` or something else?

Reply:
Yeah, `TestClientHeaders`